### PR TITLE
Make RotationTransition widget alignment configurable

### PR DESCRIPTION
**Pull request by NPKompleet**
The alignment of a RotationTransition widget is set to `Alignment.center` permanently. This proposal will help make it configurable so that users can choose any custom alignment property they desire.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/transitions.dart [hunk 1]
```diff
@@ -263,6 +264,13 @@ class RotationTransition extends AnimatedWidget {
   /// rotated v * 2 * pi radians before being painted.
   Animation<double> get turns => listenable;
 
+  /// The alignment of the origin of the coordinate system around which the
+  /// rotation occurs takes place, relative to the size of the box.
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/widgets/transitions.dart]: Nit: remove occurs (or takes place)

File: packages/flutter_tools/templates/create/lib/main.dart.tmpl [hunk 2]
```diff
@@ -10,14 +10,14 @@ import 'package:{{pluginProjectName}}/{{pluginProjectName}}.dart';
 {{/withPluginHook}}
 
 {{^withDriverTest}}
-void main() => runApp(new MyApp());
+void main() => runApp(MyApp());
```
Comment (jonahwilliams) [inline packages/flutter_tools/templates/create/lib/main.dart.tmpl]: I think you've picked up some extra changes, could you revert these?

**Review by jonahwilliams — CHANGES_REQUESTED** [review]
This will need tests. I would recommend looking in transitions_test.dart for some inspiration.

File: packages/flutter_tools/templates/create/lib/main.dart.tmpl (hunk 2 above)
Comment (NPKompleet) [reply packages/flutter_tools/templates/create/lib/main.dart.tmpl]: @jonahwilliams yeah I just noticed. I will look at all your suggestions and let you know as soon as I am done.

**Comment by NPKompleet** [comment]
Hi @jonahwilliams I just added the changes you requested. Let me know if there is any other thing you will like me to add.

**Review by jonahwilliams — APPROVED** [review]

**Pull request merged**